extensions/wasm/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# This is a generated file. Do not edit directly!
+# To re-generate, run the following command from the top level directory:
+#
+#   mvn -N cq:update-quarkus-metadata
+#
+---
+name: "Camel Wasm"
+description: "Invoke Wasm functions"
+metadata:
+  icon-url: "https://raw.githubusercontent.com/apache/camel-website/main/antora-ui-camel/src/img/logo-d.svg"
+  guide: "https://camel.apache.org/camel-quarkus/latest/reference/extensions/wasm.html"
+  categories:
+  - "integration"
+  status:
+  - "stable"
```

Comment:
As today the [component support level is experimental](https://github.com/apache/camel/blob/7676ee4dec59ac6ff6b7069c8ac42686212bd952/components/camel-wasm/pom.xml#L37), I don't recall how it maps to extensions status so wonder if this is correct

Reply:
Thanks - I'll fix it up after the CI build completes.

Reply:
Hmm, I think it should be `experimental` as well in extensions.

Reply:
It's updated now.

The maven plugins that determine the status probably need to be made smarter and interrogate the component metadata.

I can create a follow up issue for that.

Reply:
Yeah, this file could be generated by `cq-maven-plugin`.